src/components/menu/menu-gestures.ts

```diff
@@ -39,25 +45,30 @@ export class MenuContentGesture extends SlideEdgeGesture {
       // right side
       if (menu.isOpen) {
         // right side, opened
-        return true;
-
+        return this.gesture.capture();
       } else {
         // right side, closed
         if ((ev.angle > 140 && ev.angle <= 180) || (ev.angle > -140 && ev.angle <= -180)) {
-          return super.canStart(ev);
+          if (super.canStart(ev)) {
```

Comment:
This code is repeated a lot, we have to refactor before merge.
Also, it would be good if we could call: gesture.start() just once hammer starts.